src/commands/new-proj.ts

```diff
@@ -1,14 +1,22 @@
 import Vorpal from "vorpal";
-import { newRepo } from "./new-repo";
+import { newRepo } from "./new-proj/new-repo";
```

Comment:
new-repo does not necessarily need to be inside new-proj folder, it is a convenient way for creating a new repo, so the user might want to use it separately from new-proj

Reply:
refactored new-repo back under commands/